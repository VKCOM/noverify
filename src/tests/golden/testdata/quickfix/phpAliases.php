<?php
declare(strict_types = "1")
$_ = join("", []);

function test($d){
ocicollmax();
}

test(join("", []));
