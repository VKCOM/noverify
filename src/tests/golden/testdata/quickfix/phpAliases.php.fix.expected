<?php
declare(strict_types = "1")
$_ = implode("", []);

function test($d){
OCICollection::max();
}

test(implode("", []));
